Ports/dtc/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=dtc
+version=98a07006c48dc0bc3f42b3b3ce75b7f03e87e724
+workdir="dtc-${version}"
+auth_type=sha256
+files="https://github.com/dgibson/dtc/archive/${version}.zip dtc-${version}.zip 52cffaf31085cb9dce4afb91617a7017065cff7668c9a23291ad04f8c85270c4"
```

Comment:
Eh, a ZIP is technically fine, but basically everything else uses `.tar.gz`. As long as there are no files that might require their file modes to be preserved, I'll leave it up to you on whether to change it.